creators/routes_creator.py

```diff
@@ -25,4 +25,61 @@ def add_routes_to_schedule(self, schedule, data):
         route_color: # From Line
         route_text_color  # From Line
         """
-        raise NotImplementedError("Should have implemented this")
+        # Get route information
+        lines = data.get_routes()
+
+        # Loop through all lines
+        for line_ref, line in sorted(lines.iteritems()):
+
+            # Add route information
+            route = schedule.AddRoute(
+                route_id=line_ref,
+                route_type=self._get_route_type(line),
```

Comment:
I am not sure it is a good idea to get this from the config, because there might be providers with different route types like bus, tram and trains. Shouldn't this come from OSM ideally?

Reply:
I created different small functions for the different properties to, like you proposed, be able to override them in specific creators later. In these functions, I first only got the information from OSM and overrode the functions in Incofer's creator, but then later thought that it would be cool if one could specify the values in the configuration. But with your other comment, I indeed think that this is overkill and not wanted. I just pushed my old commit which is much more simple.

Reply:
I preffer getting the attributes from OSM. In the case of GTFS optional attributes as fallback we can set a default value for all providers and a warning message with help on how to fix it in OSM.  (as an enhacement, when running the script we can set a flag to ignore warnings ).  

For required attributes (I also prefer to get them from OSM and printing the error in case of)  I'll agree to have the fallback value in the config file.

Reply:
Wouldn't be better to handle the attributes in `core.osm_connector.py`?